marlowe-cli/examples/cfd/run-cfd.sh

```diff
@@ -131,15 +141,15 @@ echo "The oracle mints the role tokens."
 MINT_EXPIRES=$((TIP + 1000000))
 
 ROLE_CURRENCY=$(
-marlowe-cli util mint --testnet-magic "$MAGIC" \
+$MARLOWE_CLI_BIN -- util mint --testnet-magic "$MAGIC"                  \
                       --socket-path "$CARDANO_NODE_SOCKET_PATH"         \
                       --required-signer "$ORACLE_PAYMENT_SKEY"          \
                       --change-address  "$ORACLE_ADDRESS"               \
                       --expires "$MINT_EXPIRES"                         \
                       --out-file /dev/null                              \
                       --submit=600                                      \
                       "$ORACLE_ROLE" "$PARTY_ROLE" "$COUNTERPARTY_ROLE" \
-| sed -e 's/^PolicyID "\(.*\)"$/\1/'                                    \
+| tail -n +2 | sed -e 's/^PolicyID "\(.*\)"$/\1/'                       \
```

Comment:
I actually think we should change the `putStrLn` for most of the `marlowe-cli` commands so that one doesn't need to use `sed`: i.e., `marlowe-cli` should just print the relevant hashes (or hash#ix for transactions). This is less human readable, but it makes scripting much cleaner. However, it will break all pre-existing scripts.